src/features/surveys/components/SurveyURLCard.tsx

```diff
@@ -37,9 +37,9 @@ const SurveyURLCard = ({ isOpen, orgId, surveyId }: SurveyURLCardProps) => {
     >
       <Box display="flex" paddingBottom={2}>
         <ZUITextfieldToClipboard
-          copyText={`${process.env.NEXT_PUBLIC_ZETKIN_APP_DOMAIN}/o/${orgId}/surveys/${surveyId}`}
+          copyText={`${location.protocol}//${location.host}/o/${orgId}/surveys/${surveyId}`}
         >
-          {`${process.env.NEXT_PUBLIC_ZETKIN_APP_DOMAIN}/o/${orgId}/surveys/${surveyId}`}
+          {`${location.protocol}//${location.host}/o/${orgId}/surveys/${surveyId}`}
```

Comment:
The `orgId` in both of these places should be `survey.organization.id` (which may or may not be the organization that we're currently working within). You should be able to get it using `useSurvey()`.

See #1941 for related changes (but we refrained from making changes to the URL there to avoid merge conflicts).